### DISCUSSION TITLE
Filtering SPO Sites

### DISCUSSION THREAD

**Discussion by pratheep-r** [pnp/powershell#819] [Q&A] [answered]
I need to filter multiple SharePoint sites prefixes, can this be done in Get-PnPTenantSite
i.e.
Get-PnPTenantSite - Filter "Url -like 'X'" or "Url -like 'Y'" or "Url -like 'Z'"
It doesn't work, however is there any other way please?

**Comment by veronicageek** [comment, marked as answer]
@pratheep-r, try this:

```powershell
Get-PnPTenantSite | Where-Object {($_.Url -like "*ABC*") -or ($_.Url -like "*DEF*") -or ($_.Url -like "*XYZ*")}
```

**Reply by pratheep-r** [reply]
Oh yes, thank you.
